server/routes/viewModels/taskLists/tasks/createLicence.js

```diff
@@ -10,13 +10,13 @@ module.exports = {
     }
 
     if (approvedPremisesRequired) {
-      return approvedPremisesAddress === 'DONE' ? continueBtn('/hdc/pdf/selectLicenceType/') : null
+      return approvedPremisesAddress === 'DONE' ? continueBtn('/hdc/pdf/selectLicenceType/', 'continue') : null
```

Comment:
could push this down into the continue button too

Reply:
I can find 7 files where this is used but not that many changes in this PR.. so might be that you've missed some?

Reply:
done